pkg/ottl/ottlfuncs/README.md

```diff
@@ -2056,6 +2057,18 @@ Examples:
 - `ToKeyValueString(body)`
 - `ToKeyValueString(body, ":", ",", true)`
 
+### ToCamelCase
+
+`ToCamelCase(target)`
+
+The `ToCamelCase` Converter converts the `target` string into camel case.
```

Comment:
How about adding an example here showing the input string and the output in camel case?